packages/utils/src/stacktrace.ts

```diff
@@ -79,7 +86,7 @@ export function stripSentryFramesAndReverse(stack: StackFrame[]): StackFrame[] {
 
   // The frame where the crash happened, should be the last entry in the array
   return localStack
-    .slice(0, STACKTRACE_LIMIT)
```

Comment:
@AbhiPrasad @timfish am I right to assume that the slice should no longer be required as everything will run through the stack parser on L19? Not sure if we can remove it, or if there is some other code path that could call reverseSentryFrames fn. I kept it as the fn is exported, so better be safe than sorry here

Reply:
yeah, we can probably remove the `slice`, but perhaps someone depends on this behaviour.

We could just mark it to be removed in the next major version with a TODO.

Reply:
We do use `stripSentryFramesAndReverse` in the Electron SDK so we can combine to parsers:
https://github.com/getsentry/sentry-electron/blob/0d2eb16bf8daf9e330da85a9a823d20c80ad16ac/src/renderer/stack-parse.ts

However as long as I remember when updating, I can update the above code to drop out of the for loop at `STACKTRACE_LIMIT`

Reply:
A further search across GitHub suggests this is only used in the Electron SDK. I've opened [an issue](https://github.com/getsentry/sentry-electron/issues/653) to track this so feel free to remove the slice here and I'll fix this downstream on the next javascript sdks update!

Reply:
Actually, my point is moot because the input is a `ReadonlyArray<StackFrame>` and since our implementation is reversing the array, a copy is required regardless so that we do not end up mutating the input. 

I made the changes to use readonly type in https://github.com/getsentry/sentry-javascript/pull/7417 as it seemed reasonable and safer (the implementation was already considering it as readonly).

I think it becomes a larger change in terms of how we expect the function to be called and what safety guarantees we want, e.g. if we only expect it to be called once (which seems to be the case), then we might not care about readonly type.

I would opt to keep this as is for now and port the for loop changes to electron - this way we just avoid parsing data that will be discarded anyways and keep some extra safety in `stripSentryFramesAndReverse`